packages/types/plugins/expect.ts

```diff
@@ -0,0 +1,93 @@
+export type ExpectPluginConfig = {
+  /**
+   * @title Output format
+   */
+  outputFormat?: 'pretty' | 'json' | 'prettyError' | 'silent';
+  /**
+   * (Deprecated) Formatter
+   * Please use the `outputFormat` option instead.
+   * @deprecated true
+   * @title Formatter
+   */
+  formatter?: ExpectPluginConfig['outputFormat'];
+  /**
+   * Reports failures from expect plugin as errors
+   * in the test report.
+   * @default false
+   * @title Report failures as errors
+   */
+  reportFailuresAsErrors?: boolean;
+  /**
+   * Sets a 200 OK status code expectation for all requests.
+   * @default false
+   * @title Expect 200 by default
+   */
+  expectDefault200?: boolean;
+};
+
+export type ExpectPluginMetrics = {
+  /**
+   * Check that the response status code.
+   * If the list of status codes is provided, checks that the response
+   * status code is present in the list.
+   * https://www.artillery.io/docs/reference/extensions/expect#statuscode
+   * @title Status code
+   */
+  statusCode?: number | Array<number>;
+  /**
+   * Check that the response status code is not present
+   * in the given list.
+   * https://www.artillery.io/docs/reference/extensions/expect#notstatuscode
+   * @title Not status code
+   */
+  notStatusCode?: Array<number>;
```

Comment:
This can be `number | Array<number>` just like `statusCode`, I believe:

https://github.com/artilleryio/artillery/blob/0a58dd8f5bb096d3b0f4d3e0c4ec3076e636fb68/packages/artillery-plugin-expect/lib/expectations.js#L243-L249

Reply:
We really need to document this better. As a user, I would've never understood it can be a single value based on the [docs](https://www.artillery.io/docs/reference/extensions/expect#notstatuscode) alone.